cmd/sign/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+time crane validate --remote=sign.kontain.me/busybox
+time crane validate --remote=sign.kontain.me/busybox
```

Comment:
You can run the server locally with `BUCKET=your-gcs-bucket go run ./cmd/sign` and pull an image with `crane validate --remote=localhost:8080/busybox`

This test should also `cosign verify` the signature for the image